### ISSUE TITLE
Password protected pages direct to proudcity.com after submitting password

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#2078]
**Source**
Westhampton Beach, MVRMA and Kettering

- https://westhamptonbeach.org/police-shift-report/
- https://www.ketteringoh.org/city-council-on-boarding/
- https://www.mvrma.com/insurance-programs/

**Describe the bug**
password protected pages direct to proudcity.com after submitting password. 

**To Reproduce**
1. Go to ['...'](https://westhamptonbeach.org/police-shift-report/)
2. Enter password 
3. submit
4. Goes to my.proudcity.com 

**Expected behavior**
Users should get access to the page once they enter in the correct password. 

**Additional context**
Kettering reported that that this started last week which was a release week.

**Comment by kevindherman** [comment]
@curtismchale @jlyon could this be related to login redirects deployed recently: 

- https://github.com/proudcity/wp-proudcity/issues/1674
- https://github.com/proudcity/wp-proudcity/issues/1026

**Comment by jlyon** [comment]
@curtismchale @kevindherman I deployed a hotfix for this that doesn't redirect wp-login.php POST requests when `$_GET['action'] == 'postpass'` (see the commits in the timeline above).

@kevindherman I don't think any recent changes caused this. It was likely broken after we deployed https://github.com/proudcity/wp-proudcity/issues/1674.  The cookies appear to have a lifetime of ~14 days, so users might not have noticed this until their cookies expired and they tried to login again last week.
